tests/canary_upgrade_test.go

```diff
@@ -41,7 +41,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("[Serial][sig-operator]virt-handler canary upgrade", func() {
+var _ = Describe("[Serial][sig-operator]virt-handler canary upgrade", Labels{"Serial", "sig-operator"}, func() {
```

Comment:
Note that there is a Serial decorator that can be used instead the label
it will allow to ditch the junit-merger, at least for the e2e tests,
which now runs the parallel ones and then the serial using proprietary logic, and with the Serial decorator, same will happen but automatically

Reply:
Thanks @oshoval - I do think it should be done in a separate PR.